### PR TITLE
feat: add hm integration to nixos

### PR DESCRIPTION
**Pull request by blaggacao**

### CODE REVIEW

File: pasteurize.nix [hunk 1]
```diff
@@ -124,6 +124,12 @@
                     inherit (asserted.bee) system pkgs;
                     inherit (asserted.bee.pkgs) config; # nixos modules don't load this
                   };
+                  # seemlessly integrate hm if desired
+                  imports =
+                    []
+                    ++ l.optionals (asserted.bee ? home) [
+                      config.bee.home.nixosModules.home-manager
+                    ];
```
Comment (blaggacao) [inline pasteurize.nix:132]: I'm pretty certain that this is not enough and something like "use global packages" is required.

@gtrunsec can you inform the details about the correct configuration?

Comment (GTrunSec) [reply pasteurize.nix:132]: https://github.com/GTrunSec/hive/blob/main/nix/guangtao/nixosConfigurations/desktop/default.nix#L46

**Comment by GTrunSec** [comment]
Is it necessary to assert the homeModules ?
```
[ERROR]   stderr)        Failed assertions:
[ERROR]   stderr)        - The ‘fileSystems’ option does not specify your root file system.
[ERROR]   stderr)        - You must set the option ‘boot.loader.grub.devices’ or 'boot.loader.grub.mirroredBoots' to make the system bootable.
```

**Comment by blaggacao** [comment]
I guess a recent version of `disko` "fixed" this. Or it at least it is known upstream.

**Comment by blaggacao** [comment]
This error doesn't seem causally surfaced by any code in this PR (or repo).

If I'm mistaken, may you pin point?

**Comment by blaggacao** [comment]
@GTrunSec @neo5torm Can we merge this?

**Comment by blaggacao** [comment]
@GTrunSec Thanks for the fix!

**Pull request merged**